docs/docs/about/references/keywords/MAXIMUM_POWER.md

```diff
@@ -0,0 +1,25 @@
+# MAXIMUM_POWER
+
+[MODELS](/about/references/keywords/MODELS.md) / 
+[MAXIMUM_POWER](/about/references/keywords/MAXIMUM_POWER.md)
+
+## Description
+
+`MAXIMUM_POWER` is an optional constand maximum power (MW) that the compressor train can provide.
+## Functionality
+
+ It is an optional setting and supported for compressor train models [SIMPLIFIED_VARIABLE_SPEED_COMPRESSOR_TRAIN](/about/modelling/setup/models/compressor_modelling/compressor_models_types/simplified_variable_speed_compressor_train_model.md), [VARIABLE_SPEED_COMPRESSOR_TRAIN](/about/modelling/setup/models/compressor_modelling/compressor_models_types/variable_speed_compressor_train_model.md), [VARIABLE_SPEED_COMPRESSOR_TRAIN_MULTIPLE_STREAMS_AND_PRESSURES](/about/modelling/setup/models/compressor_modelling/compressor_models_types/variable_speed_compressor_train_model_with_multiple_streams_and_pressures.md).
+
```

Comment:
Also valid for single speed compressor trains

Reply:
SINGLE_SPEED_COMPRESSOR_TRAIN was added in MAXIMUM_POWER Functionality.